File: Kotlin/1045.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    var a = input.nextDouble();
    var b = input.nextDouble();
    var c = input.nextDouble();
    val list_sorted = listOf(a, b, c).sortedDescending();
    a = list_sorted[0];
    b = list_sorted[1];
    c = list_sorted[2];
    
    if (a >= b + c) {
        println("NAO FORMA TRIANGULO");
    }
    else if (Math.pow(a,2.0) == Math.pow(b, 2.0) + Math.pow(c, 2.0)) {
        println("TRIANGULO RETANGULO");
    }
    else if (Math.pow(a, 2.0) > Math.pow(b, 2.0) + Math.pow(c, 2.0)) {
        println("TRIANGULO OBTUSANGULO");
    }
    else if (Math.pow(a, 2.0) < Math.pow(b, 2.0) + Math.pow(c, 2.0)) {
        println("TRIANGULO ACUTANGULO");
    }
    if (a == b && b == c) {
        println("TRIANGULO EQUILATERO");
    }
    else if (a == b && b != c || a != b && b == c || a == c && c != b) {
        println("TRIANGULO ISOSCELES");
    }
}

File: Php/1045.php
<?php
[$a, $b, $c] = explode(" ", readline());
$line1_sorted = [$a, $b, $c];
rsort($line1_sorted, SORT_NUMERIC);
[$a, $b, $c] = $line1_sorted;

if ($a >= $b + $c) {
    print("NAO FORMA TRIANGULO\n");
}
else if (pow($a, 2) == pow($b, 2) + pow($c, 2)) {
    print("TRIANGULO RETANGULO\n");
}
else if (pow($a, 2) > pow($b, 2) + pow($c, 2)) {
    print("TRIANGULO OBTUSANGULO\n");
}
else if (pow($a, 2) < pow($b, 2) + pow($c, 2)) {
    print("TRIANGULO ACUTANGULO\n");
}
if ($a == $b && $b == $c) {
    print("TRIANGULO EQUILATERO\n");
}
else if ($a == $b && $b != $c || $a != $b && $b == $c || $a == $c && $c != $b) {
    print("TRIANGULO ISOSCELES\n");
}
?>